homeworks/03-data-structure/tests/BinaryTree.test.ts

```diff
@@ -0,0 +1,66 @@
+import { BinaryTree, TraverseType } from "../src/BinaryTree";
+
+describe("BinaryTree", () => {
+  const value13 = { value: 13, left: null, right: null };
+  const value14 = { value: 14, left: value13, right: null };
+  const value7 = { value: 7, left: null, right: null };
+  const value4 = { value: 4, left: null, right: null };
+  const value6 = { value: 6, left: value4, right: value7 };
+  const value1 = { value: 1, left: null, right: null };
+  const value9 = { value: 9, left: null, right: null };
+  const value3 = { value: 3, left: value1, right: value6 };
+  const value10 = { value: 10, left: value9, right: value14 };
+  const root = { value: 8, left: value3, right: value10 };
+  const tree = new BinaryTree(root);
+
+  describe("Public methods", () => {
+    it("setTree", () => {
+      const setTree = new BinaryTree(root);
+      expect(setTree.setTree(root)).toEqual(setTree);
```

Comment:
Looks strange... I doubt that this test makes sense. For example, if `setTree` method will do nothing at all, you pass this test.